modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1008,11 +1018,9 @@
 
     /**
      * Whenever read load balancing is enabled, that means 'get' requests will be distributed between primary and backup
-     * nodes if it is possible and {@link CacheConfiguration#readFromBackup} is {@code true}.
+     * nodes if it is possible and {@code CacheConfiguration#readFromBackup} is {@code true}.
```

Comment:
I suppose that this method doesn't exist, right?

Reply:
This is a link to the field.